handbook/community.md

```diff
@@ -190,37 +190,22 @@ graph TD;
 
 ## Community contributions (pull requests)
 
-The top priority when community members contribute PRs is to help the person feel engaged with
-Fleet. This means acknowledging the contribution quickly (within 1 business day) and driving to a
-resolution (close/merge) as soon as possible (may take longer than 1 business day).
+When community members contribute PRs Fleet’s top priority is to help them feel engaged. We must acknowledge contributions as soon as possible, within 1 business day. We must also drive to a resolution (close/merge) as soon as possible, though this may take longer than 1 business day.
 
 ### Process
 
-1. Decide whether the change is acceptable (see below). If this will take time, acknowledge the
-   contribution and let the user know that the team will respond. For changes that are not
-   acceptable, thank the contributor for their interest and encourage them to open an issue or
-   discuss proposed changes in the `#fleet` channel of osquery Slack before working on any more
-   code.
-2. Help the contributor make the content appropriate for merging. Ensure that the appropriate manual
-   and automated testing has been performed, changes to files and documentation are updated, etc.
-   Usually, this is best done by code review and coaching the user. Sometimes (typically for
-   customers), a Fleet team member may take a PR to completion by adding the appropriate testing and
-   code review improvements.
-3. After reviewing a PR and addressing all necessary changes, any Fleet team member may merge a 
-   community. Before merging, double-check that the CI is passing, documentation is updated, and
-   changes file is created. Please use your best judgment.
-4. Once a PR has been approved and merged, thank and congratulate the contributor, then share with the team in the `#help-promote` channel of Fleet Slack to be publicized on social media. Those who contribute to Fleet and are recognized for their contributions often become great champions for the project.
+1. Decide if the change is acceptable (see below). If this will take time, acknowledge the contribution and let the user know that the team will respond. If you choose not to approve a change, first thank the contributor for their interest. Next, encourage them to open an issue or discuss their ideas before writing more code. This can occur in the #fleet channel of osquery's Slack.
```

Comment:
"This can occur in the #fleet channel of osquery's Slack."
What can occur? They can open an issue in Slack or this conversation with the contributor? Please rewrite for clarification.

Reply:
Revised this, thanks!